setup.cfg

```diff
@@ -48,7 +48,7 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
-    # ignore benign Cython warnings on ndarray size
+    #ignore::DeprecationWarning:
```

Comment:
why was this changed?

Reply:
It's the exact same as master so effectively, nothing changed.